content/jobs.njk

```diff
@@ -0,0 +1,59 @@
+---
+title: Jobs
+---
+{% extends "layouts/base.layout.njk" %}
+
+{% block content %}
+    <section class="container-xxl my-3">
+        <h1 id="bounties" class="fw-bold"><a href="#bounties">Bounties</a></h1>
+        <p>
+            The <span class="fw-bold"><a href="/foundation/">MonoGame Foundation</a></span> uses a bounty system to propose paid
+			missions to improve the project.
+        </p>
+		<p>
+            The list of all available bounties is available on <a href="https://github.com/MonoGame/MonoGame/issues/8120">the
+			project's Github repository</a>.
+        </p>
+    </section>
+
+    <section class="container-xxl mb-3">
+        <h1 id="how" class="fw-bold"><a href="#how">How do bounties work?</a></h1>
+        <p>
+            If you would like to propose your service to fulfill a bounty, please send us your references, a proposed schedule to
+			complete the bounty, and a quote by e-mail (please indicate which bounty you are applying to): <a href="mailto:biz@monogame.net">biz@monogame.net</a>
+        </p>
+		<p>
+            The Foundation board will review your application during the next board meeting (happening every second Wednesday of
+			each month; and providing that you sent your application at least 48 hours ahead of the next meeting). The board will
+			then let you know of the decision. If the decision is favorable, a mission contract will be proposed to you.
+        </p>
+		<p>
+			The Foundation board will consider your expertise and proposed schedule over your rate. Hence, bounties are not bids for
+			the cheapest. The board will look for the most suitable application.
+		</p>
+    </section>
+
+	<section class="container-xxl mb-3">
+        <h1 id="requirements" class="fw-bold"><a href="#requirements">Requirements to apply for bounties</a></h1>
+        <p>
+            Before you apply for fulfilling a bounty, please note the following requirements:
+        </p>
+		<ul>
+		  <li>You must be a registered business or a registered self-employed individual.</li>
+		  <li>Your country of registration must not be under commercial embargo from the United States of America.</li>
+		  <li>Your bank account must not be from a bank that is under commercial embargo from the United States of America.</li>
+		  <li>
+		      Your bank account must be able to receive payments in $USD, or payments in
+			  <a href="https://wise.com/help/articles/2571942/what-countriesregions-can-i-send-to?id=2571942&slug=what-countries-can-i-send-to">
+			  any currency supported by Wise</a> (a Wise account isn't required).
+		  </li>
+		</ul>
+		<p>
+            We can unfortunately not consider applications that don't meet these requirements.
+        </p>
+    </section>
+    
+{% endblock %}
+
+{% block scripts %}
```

Comment:
Unless you have an active script you need to run on the page, you can remove the scripts block. 
I may have accidentally left it on other pages while developing, but it's not needed unless you need a page specific script.